src/mon/MonCommands.h

```diff
@@ -449,6 +449,11 @@ COMMAND("fs mirror peer_remove "
 	"name=fs_name,type=CephString "
 	"name=uuid,type=CephString ",
 	"remove a mirror peer for a ceph filesystem", "mds", "rw")
+COMMAND("fs rename "
+	"name=fs_name,type=CephString "
+	"name=new_fs_name,type=CephString,goodchars=[A-Za-z0-9-_.] "
```

Comment:
Can you pull `[A-Za-z0-9-_.]` out into a macro `#define` and fix the other MonCommands to use that define too. Something like:

````
#define FS_NAME_GOODCHARS "[A-Za-z0-9-_.]"

...
     "name=new_fs_name,type=CephString,goodchars=" FS_NAME_GOODCHARS
...
````

Reply:
Done.